api.bs

```diff
@@ -450,20 +454,98 @@ how to handle weeks in which the [=privacy budget=] is insufficient,
 and (optionally) how to process any additional parameters that might be used.
 
 
-### Last Touch Attribution ## {#logic-last-touch}
+### Last Touch Attribution ### {#logic-last-touch}
 
 The <dfn enum-value for=PrivateAttributionLogic>last-touch</dfn> [=attribution logic=]
 indicates that the browser should select
 the last impression that matches the [[#logic-matching|common matching logic]].
 The entire [=conversion value=] is allocated to the histogram bucket
 that was saved with the impression.
 
+Last touch attribution does not select any impression
+that was saved during a week
+that does not have sufficient [=privacy budget=].
+If impressions match from a week
+that does not have enough [=privacy budget=],
+impressions are not matched for any preceding weeks.
+That is, once a week has a matching impression
+and insufficient budget,
+the process will set a value of zero for all histogram buckets.
+
+To <dfn ignore>fill a histogram using last-touch attribution</dfn>,
+given |options|:
```

Comment:
This method of writing code is super, super annoying.  This is far from being ready, but it has the bones that I think we need.